central/detection/alertmanager/alert_manager_impl.go

```diff
@@ -317,6 +318,8 @@ func (d *alertManagerImpl) mergeManyAlerts(
 			mergedAlert := mergeAlerts(matchingOld, alert)
 			if mergedAlert != matchingOld && !proto.Equal(mergedAlert, matchingOld) {
 				updatedAlerts = append(updatedAlerts, mergedAlert)
+			} else {
+				metrics.IncDedupedAlerts(alert.GetLifecycleStage(), alert.GetPolicy().GetName())
```

Comment:
I'm not super familiar with prometheus and how it handles arbitrarily large tags, but policy name can be long. What about id? Can always correlate later by pulling policies from store/db.

Reply:
My concern is if someone creates a new policy, then we won't know the content